pyrogram/client/client.py

```diff
@@ -74,7 +78,8 @@ class Client:
     INVITE_LINK_RE = re.compile(r"^(?:https?:\/\/)?t\.me\/joinchat\/(.+)$")
     DIALOGS_AT_ONCE = 100
 
-    def __init__(self, session_name: str, test_mode: bool = False):
+    def __init__(self, config_file: str = "config.ini", session_name: str = None, test_mode: bool = False):
```

Comment:
Maybe we should keep `session_name` as the first parameter so you don't have to explicitly pass a kwarg every time.

Reply:
That is ok for me. As you like it best.

Reply:
It just seemed me conceptually right to have the config file first :smile: